docs/_posts/2014-08-31-sor.markdown

```diff
@@ -78,6 +77,16 @@ protected void initialize(Configuration configuration, Environment environment)
 }
 ```
 
+REST calls
+----------
+
+The Java client library is a convenience; EmoDB exposes a RESTful API for use by any client.  Examples for both the Java
+and REST calls are provided below.
+
+Note that the REST API requires [API keys]({{ site.baseurl }}/security) and that the Java client includes these
+in all requests automatically.  For clarity the API key header is not included each REST example below, but in a
+properly secured system you would need to add it to each request.
```

Comment:
To my previous point, in the quick tutorial section. I like this note to be there too, and not have to worry about adding security header.